app-misc/prog-express/prog-express-3.7.8.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop udev unpacker
+
+DESCRIPTION="A modern and intuitive control software for the Batronix USB programming devices"
+HOMEPAGE="https://www.batronix.com"
+SRC_URI="amd64? ( https://www.batronix.com/exe/Batronix/Prog-Express/deb/${P}-1.amd64.deb )
+	x86? ( https://www.batronix.com/exe/Batronix/Prog-Express/deb/${P}-1.i386.deb )"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="prog-express"
+SLOT="0"
+
+RDEPEND="dev-db/sqlite:3
+	dev-dotnet/gtk-sharp:2
+	dev-dotnet/libgdiplus
+	dev-lang/mono
+	dev-lang/mono-basic
+	virtual/libusb:1
+	virtual/udev"
+
+RESTRICT="strip"
+
+S="${WORKDIR}"
+
+DOCS=( "usr/share/doc/prog-express/changelog.gz" "usr/share/doc/prog-express/manuals" )
```

Comment:
Installing gzipped docs is not fine. Unpack it.

Reply:
Whoops, thanks.